src/js/game/systems/item_ejector.js

```diff
@@ -294,7 +300,18 @@ export class ItemEjectorSystem extends GameSystemWithFilter {
             }
         }
 
-        return false;
+        // it could be a custom mod case
+        // let's check if it is any of them
+        var any = false; // so that all matching components can process it
+        for (let id in MOD_ITEM_FILTERS) {
+            if (receiver.components[id]) {

```

Comment:
feel like item filters should be handled by a single configurable component, for performance (you already know which filters you need to check); maybe this breaks ECS though

Reply:
gonna be honest i have no idea what ECS stands for. went ahead and implemented the component anyway. thanks for the suggestion

Reply:
im like 85% sure it doesn't break ECS now that i know what it means.